beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -13,6 +20,73 @@ const WDTerritory: React.FC<WDTerritoryProps> = function ({
   territoryMapData,
 }): React.ReactElement {
   const theme = useTheme();
+  const dispatch = useAppDispatch();
+
+  const [territoryFill, setTerritoryFill] = React.useState<string | undefined>(
+    territoryMapData.fill,
+  );
+
+  const [territoryFillOpacity, setTerritoryFillOpacity] = React.useState<
+    number | undefined
+  >(undefined);
+
+  const [territoryStrokeOpacity, setTerritoryStrokeOpacity] = React.useState(1);
+
+  const commands = useAppSelector(
+    (state) => state.game.commands.territoryCommands[territoryMapData.name],
+  );
+
+  const {
+    user: { member },
+  } = useAppSelector(gameOverview);
+  const userCountry = countryMap[member.country];
+
+  const deleteCommand = (key) => {
+    dispatch(
+      gameApiSliceActions.deleteCommand({
+        type: "territoryCommands",
+        id: territoryMapData.name,
+        command: key,
+      }),
+    );
+  };
+
+  if (commands && commands.size > 0) {
```

Comment:
Would only checking for `commands.size > 0` work here?

Reply:
I don't think so because commands may be undefined. I suppose I can use `commands?.size`